tests/contrib/database/test_mysql.py

```diff
@@ -5,16 +5,49 @@
 import sys
 import shutil
 import tempfile
+import six
+
+
+from charmhelpers.fetch import (
+    apt_install,
+    apt_update,
+    filter_installed_packages,
+)
 
 
 sys.modules['MySQLdb'] = mock.Mock()
 from charmhelpers.contrib.database import mysql  # noqa
 
 
+try:
+    import MySQLdb
+except ImportError:
+    apt_update(fatal=True)
+    if six.PY2:
+        apt_install(filter_installed_packages(['python-mysqldb']), fatal=True)
+    else:
+        apt_install(filter_installed_packages(['python3-mysqldb']), fatal=True)
+    import MySQLdb
```

Comment:
Why are you trying to install MySQLdb -- it's mocked out above in 18 via `sys.modules`?

Reply:
I believe tests in the previous PR were failing when this part was missing. Let me try to re-patch.

Reply:
@ajkavanagh so is it fine to leave it as it is, or is there any other way to refer to MySQLdb?

Reply:
This simply should NOT be necessary for a unit test.  The sys.modules[...] on line 18 mocks in the whole of the module.